packages/driver/src/httpScram.ts

```diff
@@ -24,6 +24,31 @@ function b64ToUtf8(str: string): string {
   return utf8Decoder.decode(decodeB64(str));
 }
 
+/**
+ * Parses SCRAM-SHA-256 authentication parameters from a string.
+ * @param authParamsString a string of comma-separated key-value pairs
+ * @returns an object with `sid` and `data` properties, decoded string or `null` when not present
+ */
+function parseSidAndData(paramsStr: string): {
```

Comment:
Non-blocking naming thought: I wonder if `parseScramAttrs` is maybe a bit more descriptive since it gives you so context as to _what_ `sid` and `data` are?